webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -76,7 +79,7 @@ export const ComparisonTableMultiCell: React.FC<{
             setCurrentStep(Number(event.target.value))
           }}
         />
-        <p>{currentStep}</p>
+        <p>{selectedImg?.ind}</p>
```

Comment:
[Q] Can this actually be undefined? Do we need the optional chain and should we show something instead of undefined if that can actually happen?

Reply:
If I'm understanding your question, should not be possible for `ind` to be undefined. Just a typo on my part :)